xml/dhcp_vendor_class.xml

```diff
@@ -144,6 +144,17 @@
     <param pos="0" name="os.vendor" value="Polycom"/>
   </fingerprint>
 
+  <fingerprint pattern="^Polycom-SSIP([4-7]000)$">
+    <description>Polycom SoundStation IP Phone</description>
+    <example hw.product="6000" hw.model="6000">Polycom-SSIP6000</example>
+    <example hw.product="7000" hw.model="7000">Polycom-SSIP7000</example>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="0" name="hw.vendor" value="Polycom"/>
+    <param pos="0" name="hw.family" value="SoundStation IP"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="{hw.model}"/>
```

Comment:
`hw.product` should be `{hw.family} {hw.model}` in order to match existing CPE values.
```
h:polycom:soundstation_ip_4000
h:polycom:soundstation_ip_5000
h:polycom:soundstation_ip_6000
h:polycom:soundstation_ip_7000
```

Reply:
Done